interface/billing/sl_eob_search.php

```diff
@@ -1,849 +1,832 @@
+      //echo $file;
+      $content = ob_get_clean();
+      // Fix a nasty html2pdf bug - it ignores document root!
+      global $web_root, $webserver_root;
+      $i = 0;
+      $wrlen = strlen($web_root);
+      $wsrlen = strlen($webserver_root);
+      while (true) {
+        $i = stripos($content, " src='/", $i + 1);
+        if ($i === false) break;
+        if (substr($content, $i+6, $wrlen) === $web_root &&
+          substr($content, $i+6, $wsrlen) !== $webserver_root) {
+          $content = substr($content, 0, $i + 6) . $webserver_root . substr($content, $i + 6 + $wrlen);
         }
       }
-
-      // Recompute age at each invoice.
-    $stmt['age'] = round((strtotime($today) - strtotime($stmt['duedate'])) / (24 * 60 * 60));
-
-      $invlines = ar_get_invoice_summary($row['pid'], $row['encounter'], true);
-      foreach ($invlines as $key => $value) {
-        $line = array();
-        $line['dos']     = $svcdate;
-        if ($GLOBALS['use_custom_statement']) {
-          $line['desc']    = ($key == 'CO-PAY') ? "Patient Payment" : $value['code_text']; 
-      } else {
-        $line['desc']    = ($key == 'CO-PAY') ? "Patient Payment" : "Procedure $key";
-        } 
-        $line['amount']  = sprintf("%.2f", $value['chg']);
-        $line['adjust']  = sprintf("%.2f", $value['adj']);
-        $line['paid']    = sprintf("%.2f", $value['chg'] - $value['bal']);
-        $line['notice']  = $duncount + 1;
-        $line['detail']  = $value['dtl'];
-        $stmt['lines'][] = $line;
-        $stmt['amount']  = sprintf("%.2f", $stmt['amount'] + $value['bal']);
-        $stmt['ins_paid']  = $stmt['ins_paid'] + $value['ins'];
-      }
-
-      // Record that this statement was run.
-      if (! $DEBUG && ! $_POST['form_without']) {
-        sqlStatement("UPDATE form_encounter SET " .
-          "last_stmt_date = '$today', stmt_count = stmt_count + 1 " .
-          "WHERE id = " . $row['id']);
-      }
-      if ($_POST['form_portalnotify']) {
-        if( ! is_auth_portal($stmt['pid']) ){
-            $alertmsg = xlt('Notification FAILED: Not Portal Authorized');
-            break;
+      $pdf2->WriteHTML($content);
+      $temp_filename = $STMT_TEMP_FILE_PDF;
+      $content_pdf = $pdf2->Output($STMT_TEMP_FILE_PDF,'F');
+    } else {
+    $pdf = new Cezpdf('LETTER');//pdf creation starts
+    $pdf->ezSetMargins(45,9,36,10);
+    $pdf->selectFont($GLOBALS['fileroot'] . "/library/fonts/Courier.afm");
+    $pdf->ezSetY($pdf->ez['pageHeight'] - $pdf->ez['topMargin']);
+    $countline=1;
+    $file = fopen($file_to_send, "r");//this file contains the text to be converted to pdf.
+      while(!feof($file)) {
+      $OneLine=fgets($file);//one line is read
+       if(stristr($OneLine, "\014") == true && !feof($file))//form feed means we should start a new page.
+        {
+          $pdf->ezNewPage();
+          $pdf->ezSetY($pdf->ez['pageHeight'] - $pdf->ez['topMargin']);
+          str_replace("\014", "", $OneLine);
         }
```

Comment:
Real changes start here I suppose.

Reply:
I can't solve the merge conflict. I don't know what is causing the problem.

Reply:
I think it's good now.